File: Academy/webapi/webapi/Models/DTOes/SignUpUser.cs
namespace webapi.Models;

public class SignUpUser
{
    public string Name { get; set; }
    public string Surname { get; set; }
    public int Age { get; set; }
    public string Email { get; set; }
    public string Password { get; set; }
    public string ConfirmPassword { get; set; }
}

File: Academy/academyui/src/models/SignUpUser.js
class SignUpUser {
  constructor(name = '', surname = '', age = '', email = '', password = '', confirmPassword = '') {
    this.Name = name;
    this.Surname = surname;
    this.Age = age;
    this.Email = email;
    this.Password = password;
    this.ConfirmPassword = confirmPassword;
  }
}

export default SignUpUser;